### ISSUE TITLE
invalid metric name "redis_cluster_stats_messages_auth-ack_received"

### ISSUE THREAD

**Issue by azr** [oliver006/redis_exporter#93]
howdy, after a bad period on our redis cluster the exporter stopper working as we started seeing:

```
invalid metric name "redis_cluster_stats_messages_auth-ack_received"
```

There's also
```
redis_cluster_stats_messages_auth-req_sent
```

**Comment by oliver006** [comment]
I tried with a 3-node cluster using Redis 3.0.6 and 4.0.1 but couldn't reproduce this - can you provide more information about your setup please?

**Comment by azr** [comment]
We have 3 redis-master that each have one slave.
Version is 4.0.1.

When you set the cluster up, this doesn't happen.
Our datacenter started having connectivity issues & failover started coming in place.
We thought the service was down for a minute, turns out that dash was making metrics output crash.

PS: I have deployed that image which eludes metrics with a dash for now, works fine :) .

**Comment by jdvr** [comment]
I have exactly the same problem after some nodes of the cluster restart.

**Comment by oliver006** [comment]
I finally managed to reproduce this.

But the fix in #94 works for me so I'll go ahead and get that merged, that should resolve the issue.

**Comment by oliver006** [comment]
https://github.com/oliver006/redis_exporter/releases/tag/v0.12.1 should fix this - please report back if you have a chance.
